src/lib/transform.js

```diff
@@ -194,6 +199,7 @@ const arTransformers = [
   'endDate',
   'startDate',
   transformRelatedModel('activityRecipients', 'name'),
+  transformGrantModel('programTypes'),
```

Comment:
pretty minor but can we keep the columns in the same order? I'd hate for someone to have formulas setup in excel that depend on the order of the columns. Actually I'm not even sure how the header is set, so these may need to stay in the same order so values match the header.